<?xml version="1.0" encoding="utf-8" ?>
<launch>
    <arg name="bagPath" default="/seerep/seerep-data/lero/2023-09-08-13-34-26.bag"/>
    <arg name="hdf5FolderPath" default="/seerep/seerep-data/test"/>
    <arg name="projectFrameId" default="map"/>
    <arg name="projectName" default="testproject"/>
    <arg name="topicImage" default="/camera_c/color/image_rect_color"/>

    <node name="seerep_ros_communication_rosbagConverterEval" pkg="seerep_ros_communication" type="seerep_ros_communication_rosbagConverterEval" output="screen">
        <rosparam param="bagPath" subst_value="True">$(arg bagPath)</rosparam>
        <rosparam param="hdf5FolderPath" subst_value="True">$(arg hdf5FolderPath)</rosparam>
        <rosparam param="projectFrameId" subst_value="True">$(arg projectFrameId)</rosparam>
        <rosparam param="projectName" subst_value="True">$(arg projectName)</rosparam>
        <rosparam param="topicImage" subst_value="True">$(arg topicImage)</rosparam>
    </node>
</launch>
